.ci/Jenkinsfile_visualBaseline

```diff
@@ -0,0 +1,29 @@
+#!/bin/groovy
+
+library 'kibana-pipeline-library'
+kibanaLibrary.load()
+
+stage("Kibana Pipeline") { // This stage is just here to help the BlueOcean UI a little bit
+  timeout(time: 120, unit: 'MINUTES') {
+    timestamps {
+      ansiColor('xterm') {
+        catchError {
+          parallel([
+            'kibana-oss-agent': kibanaPipeline.withWorkers('kibana-oss-tests', { kibanaPipeline.buildOss() }, [
```

Comment:
```
kibanaPipeline.withWorker('kibana-oss-tests', 'linux && immutable') {
  kibanaPipeline.buildOss()
  kibanaPipeline.getPostBuildWorker('visualRegression', { runbld('./test/scripts/jenkins_visual_regression.sh', 'Execute kibana-visualRegression') })(1)
}
```

Something like this might work without spinning up a really big machine for one task